nix/hydra/shell.nix

```diff
@@ -135,6 +135,7 @@ let
       cardano-node.cardano-node
       cardano-node.cardano-cli
       hsPkgs.hydra-node.components.exes.hydra-node
+      hsPkgs.hydra-cluster.components.exes.hydra-cluster
```

Comment:
Not needed.. right? The `exes` shell includes it already. Or where you confused which one to pick?

Reply:
Leftover from smoke tests CI debug...ooops